src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2530 @@
+    - `small` - a slab allocator. The allocator repeatedly uses a memory
+      block to allocate objects of the same type. Note that this allocator is
+      prone to unresolvable fragmentation on specific workloads, so you can
+      switch to `system` in such cases.
+]])
+
+I['memtx.max_tuple_size'] = format_text([[
+    Size of the largest allocation unit for the memtx storage engine in bytes.
+    It can be increased if it is necessary to store large tuples.
+]])
+
+I['memtx.memory'] = format_text([[
+    The amount of memory in bytes that Tarantool allocates to store tuples.
+    When the limit is reached, `INSERT` and `UPDATE` requests fail with the
+    `ER_MEMORY_ISSUE` error. The server does not go beyond the `memtx.memory`
+    limit to allocate tuples, but there is additional memory used to store
+    indexes and connection information.
+]])
+
+I['memtx.min_tuple_size'] = format_text([[
+    Size of the smallest allocation unit in bytes. It can be decreased if
+    most of the tuples are very small.
+]])
+
+I['memtx.slab_alloc_factor'] = format_text([[
+    The multiplier for computing the sizes of memory chunks that tuples
+    are stored in. A lower value may result in less wasted memory depending
+    on the total amount of memory available and the distribution of item sizes.
+]])
+
+I['memtx.slab_alloc_granularity'] = format_text([[
+    Specify the granularity in bytes of memory allocation in the small
+    allocator. The `memtx.slab_alloc_granularity` value should meet the
+    following conditions:
+
+    - The value is a power of two.
+    - The value is greater than or equal to 4.
+
+    Below are few recommendations on how to adjust the
+    `memtx.slab_alloc_granularity option`:
+
+    - If the tuples in space are small and have about the same size, set the
+      option to 4 bytes to save memory.
+    - If the tuples are different-sized, increase the option value to allocate
+      tuples from the same `mempool` (memory pool).
+]])
+
+I['memtx.sort_threads'] = format_text([[
+    The number of threads from the thread pool used to sort keys of secondary
+    indexes on loading a `memtx` database. The minimum value is 1, the maximum
+    value is 256. The default is to use all available cores.
+]])
+
+-- }}} memtx configuration
+
+-- {{{ metrics configuration
+
+I['metrics'] = format_text([[
+    The `metrics` section provides the ability to collect and expose
+    Tarantool metrics (e.g., network, cpu, memtx and others).
+]])
+
+I['metrics.exclude'] = format_text([[
+    An array containing groups of metrics to turn off. The array can
+    contain the same values as the `exclude` configuration parameter
+    passed to `metrics.cfg()`.
+]])
+
+I['metrics.exclude.*'] = 'Array of groups of metrics.'
+
+I['metrics.include'] = format_text([[
+    An array containing groups of metrics to turn on. The array can
+    contain the same values as the `exclude` configuration parameter
+    passed to `metrics.cfg()`.
+]])
```

Comment:
Should we say 'as the `include` <...> passed to `metrics.cfg()`', because we're describing the `metrics.include` option?

Reply:
Yes, it's a typo. Fixed.
```lua
I['metrics.include'] = format_text([[
    An array containing groups of metrics to turn on. The array can
    contain the same values as the `include` configuration parameter
    passed to `metrics.cfg()`.
]])
```